README.md

```diff
@@ -1,3 +1,27 @@
 # Self_reporting Bot
 
 If you send statistics to this bot, then it will save them. See https://github.com/deltachat/deltachat-android/issues/2909.
+
+## Install
+
+To install, open your terminal and run:
+
+```sh
+pip install git+https://github.com/deltachat/self_reporting_bot.git
+```
+
+## Usage
+
+Configure the bot:
+
+```sh
+telemetrybot init bot@example.com PASSWORD
```

Comment:
Is there any reason for calling it `telemetrybot`, not `self_reporting_bot`?

Reply:
Just because the later is a rather long and weird name for a program/command, isn't the bot collecting statistics/telemetry? self-reporting is a bit vague 

But if you want the command to be like that I can change it here and in the toml file where the command name is defined

Reply:
Another ideas for the command name:
 srbot

Reply:
the word "telemetry" has a quite bad conotation as often used without knowlege of the user (eg. by hiding in long eula, "calling home"). also, often the data are much more and include much fine tracking, eg. mouse movements, how long you wait until
you press a button

the wording "self reporting" seems to be more fitting

Reply:
Yes, we can use some shorter form like `reportingbot`, `repbot` or `srbot`, even though for me there would not be any need to abbreviate it since it's not a command line tool that will be used often. I agree with @r10s on why we shouldn't use `telemetry`.